about.html

```diff
@@ -0,0 +1,122 @@
+            <div class="line"></div>
+            <p class="mt-4 font-light px-6">
+                This year's WWE International Rendezvous logo was decided using an AI wich created the logo using renewable resources
+            </p>
+            <div class="border border-zinc-200 mt-12 py-8 px-10">
+                <p class="font-bold text-2xl">WWE International Rendezvous</p>
+            </div>
+
+        </section>
+
+        <!-- past section -->
+
+        <section class="py-16 text-center" id="past">
+            <h2 class="text-center text-3xl logo-title pt-8">Here we will put the photos of previous WWE International Rendezvous</h2>
+            <div class="line"></div>
+            <p class="mt-8 font-light px-6">This year's will be the first event of its kind, so you'll have a chance to be on our pics too!</p>
+            <div class="flex flex-col md:flex-row gap-x-2 gap-y-4 mt-12 mx-8 md:mx-6">
+                <div class="flex flex-col justify-center items-center past1 overlay-past">
+                    <h2 class="past-date text-4xl">2023</h2>
+                    <h3 class="past-desc font-bold">1st WWE International Rendezvous</h3>
+                </div>
+                <div class="flex flex-col justify-center items-center past2 overlay-past">
+                    <h2 class="past-date text-4xl">2024</h2>
+                    <h3 class="past-desc font-bold">2nd WWE International Rendezvous</h3>
+                </div>
+            </div>
+        </section>
+
+        <!-- footer -->
+
+        <footer class="bottom-0 w-full py-8 flex items-center justify-center bg-[var(--bg-color)]">
+
+            <h3 class="text-xl md:text-2xl font-bold px-4 md:pr-16 footer-title text-white">WWE International Rendezvous</h3>
+            <div>
+                <p class="text-white">Original Design by Cindy Shin in Behance</p>
+                <p class="text-white">Some Rights Reserved</p>
+            </div>
+        </footer>
```

Comment:
- [ OPTIONAL ] I highly recommend to add the partner section above the footer in your about page in the mobile version since it is there in the mobile version so kindly add it,🚀💯 I am assigning it as optional because the previous reviewer didn't mention it but it still important to fix it 💯🔥🔥💪🏻

- [ ] And please make sure to change thje background color for the last footer in the mobile version about page 🔥🔥
- Your 
![image](https://user-images.githubusercontent.com/58553711/233235291-c0b34a03-35f0-4547-aa97-d9b2213b1fef.png)

- Given
![image](https://user-images.githubusercontent.com/58553711/233235318-1edbd1ff-374d-4c9b-917a-25005cf03b01.png)